recipes/openssl/openssl_1.0.2g.oe

```diff
@@ -1,8 +1,5 @@
 require openssl.inc
 
-# Parallel make is simply broken for now.
-PARALLEL_MAKE = ""
```

Comment:
for the record, I will happily accept removing disabling PARALLEL_MAKE if it has been shown to work on the oe-lite buildbot. It simply sucks that if someone finds a (maybe?) parallel make issue using $my-special-j-setting it ends up being disabled for everyone.

that said, when there is a patch, of course we chose that route